### DISCUSSION TITLE
Nazwy załadowanych przez sprzedawcę załączników

### DISCUSSION THREAD

**Discussion by tomasz-grzesik** [allegro/allegro-api#10284] [Suggestions - resolved]
Przydałaby się możliwość pobrania nazw załadowanych przez sprzedawcę załączników.
Dla podanego uuid, aby metoda zwracała nazwę oryginalnie załadowanego pliku.

**Comment by MaciejFrackowiak** [comment]
Dziękuję za sugestię. Jeśli zdecydujemy się na wprowadzenie takiej możliwości, będziemy informować na naszej stronie i forum.

**Comment by MaciejFrackowiak** [comment]
@tomasz-grzesik Dzisiaj wdrożyliśmy nowy zasób odpowiedzialny za pobieranie szczegółów danego załącznika - więcej w [newsie](https://github.com/allegro/allegro-api/issues/10568).

**Reply by tomasz-grzesik** [reply]
> @tomasz-grzesik Dzisiaj wdrożyliśmy nowy zasób odpowiedzialny za pobieranie szczegółów danego załącznika - więcej w [newsie](https://github.com/allegro/allegro-api/issues/10568).

Dziękuję za powiadomienie i szybkie działanie.
